internal/jet/column_list.go

```diff
@@ -39,6 +41,21 @@ func (cl ColumnList) Except(excludedColumns ...Column) ColumnList {
 	return ret
 }
 
+// As will create new projection list where each column is wrapped with a new table alias.
+// tableAlias should be in the form 'name' or 'name.*', or it can also be an empty string.
+// For instance: If projection list has a column 'Artist.Name', and tableAlias is 'Musician.*', returned projection list will
+// have a column wrapped in alias 'Musician.Name'. If tableAlias is empty string, it removes existing table alias ('Artist.Name' becomes 'Name').
+func (cl ColumnList) As(tableAlias string) ProjectionList {
+	if tableAlias > "" {
```

Comment:
Why `>`?

Reply:
It's equivalent to `!=`, no particular reason. Will change to `!=`